FirebaseInAppMessaging/Swift/Tests/Integration/FIAMSwiftUI/Shared/ModalInAppMessageView.swift

```diff
@@ -62,7 +62,7 @@ struct ModalInAppMessageView: View {
   @ViewBuilder
   func dismissButton(modalMessage: InAppMessagingModalDisplay,
                      delegate: InAppMessagingDisplayDelegate) -> some View {
-    if let _ = modalMessage.actionButton, modalMessage.actionURL != nil {
+    if let _ = modalMessage.actionButton, modalMessage.actionURL == nil {
```

Comment:
nit: Though you didn't originally write the code this way, can you fix the check here so it's stylistically consistent?
```swift
if modalMessage.actionButton != nil && modalMessage.actionURL == nil
```

Reply:
I've changed the style, and also made it a little more correct